scripts/newinstall.sh

```diff
@@ -99,12 +117,11 @@ if [[ "$cont_flag" = false ]]; then
 	fi
 fi
 
-
-##########	Offer to get git if it's too old
+##########  Discuss the state of Git.
 
 if true; then
 	if hash git 2>/dev/null; then
-		GITVERNUM=$(git --version | cut -d" " -f 3)
+		GITVERNUM=$(git --version | cut -d\  -f 3)
```

Comment:
This is soooo easy to accidentally mess up (change two whitespaces into one, even by accident), plus newbies don't know how to read it. That's why I really prefer quoted spaces everywhere.